src/components/Table/types/TableTypes.ts

```diff
@@ -46,7 +46,7 @@ export type TableBodyProps<T extends AnyRecord> = {
 
 export interface IFilterContext {
 	allowColumnFilter: boolean;
-
+	canDrag: boolean;
```

Comment:
I think that a better name would be `draggable`. [MUI uses "rowReordering"](https://mui.com/x/react-data-grid/row-ordering/), so I'm open to ideas, but `canDrag` doesn't feel properly descriptive to me. 
```suggestion
	draggable: boolean;
```